### PR TITLE
docs: updated documentation

### PR DESCRIPTION
**Pull request by mnugter**

### CODE REVIEW

File: packages/core/documentation/Core/transformers.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 # Transformers
 
-A transformer converts the message `format` from the backend to the Hyral and the other way around.
+A transformer converts the message *format* from the backend to the Hyral and the other way around.
```
Comment (barthje) [inline packages/core/documentation/Core/transformers.md]: this reads a bit funnily. "Converts to the Hyral". Converts to the hyral format?

File: packages/core/documentation/Guides/quick-start.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 ## Configuration
 
 ```javascript
 // create a configuration file (hyral.js for example) containing something like this.
 // =========================================================================
 import axios from 'axios';
-import HttpConnector from '@hyral/core/lib/Connector/HttpConnector';
-import repositoryManager from '@hyral/core/lib/Resource/repositoryManager';
-import jsonApi from '@hyral/json-api';
+import HttpConnector from '@hyral/core/lib/Connector/HttpConnector';import repositoryManager from '@hyral/core/lib/Resource/repositoryManager';import jsonApi from '@hyral/json-api';
```
Comment (barthje) [inline packages/core/documentation/Guides/quick-start.md]: the imports are all on the same line, should be on new lines

**Comment by mnugter** [comment]
Link to the index added!

File: packages/vue/src/Module/createStoreModule.js [hunk 3]
```diff
@@ -18,17 +18,17 @@ const createStoreModule = (repository, store) => ({
   },
 
   getters: {
-    resource: state => id => state.resources[id] ? Resource.fromState(
+    resource: state => id => (state.resources[id] ? Resource.fromState(
```
Comment (barthje) [inline packages/vue/src/Module/createStoreModule.js]: You've changed tests and code in this branch, it should be in a seperated branch and PR

Comment (mnugter) [reply packages/vue/src/Module/createStoreModule.js]: yup sorry, won't do that next time ;)

**Pull request merged**